tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -822,6 +828,18 @@ main()
 
 	init="${ROOTFS_DIR}/sbin/init"
 	setup_rootfs
+
+	if [ "${VARIANT}" = "nvidia-gpu" ]; then
```

Comment:
According to this line 832 here, it seems there have been some changes when creating the GPU rootfs, right? It needs ENV VARIANT to indicate it is a gpu branch and do setup nvidia gpu rootfs with two stages.  what I miss here? I can't find how to set this ENV.

Reply:
This is set when executing rootfs.sh with the proper VARIANT. See the modified call to the rootfs.sh script.